package.json

```diff
@@ -46,7 +46,7 @@
     "setup-dev": "yarn dev-build && yarn rm-dev-link && yarn link-dev && yarn rm-aa-dev-link && yarn link-aa-dev",
     "split-e2e-tests": "yarn ts-node ./scripts/split-e2e-tests.ts && git add .circleci/config.yml",
     "test-changed": "nx affected --target=test",
-    "test-ci": "nx run-many --target=test --all -- --ci --runInBand",
+    "test-ci": "nx run-many --parallel=4 --target=test --all -- --ci --runInBand",
```

Comment:
why run parallel in NX but runInBand at the jest layer?

Reply:
That comes from this PR https://github.com/aws-amplify/amplify-cli/pull/11917 .
I.e. using both nx and jest parallelization led to spin-locks.
In this PR we decrease default 8 parallelization to 4 for that single job due to slightly higher memory pressure (the alternative was to bump instance size)

Reply:
the default is here for reference https://github.com/aws-amplify/amplify-cli/blob/78590b8d4face7bbc0b174337b77b118affc525b/nx.json#L9